dev/TeachingTip/TeachingTip.h

```diff
@@ -293,4 +293,6 @@ class TeachingTip :
 
     //Ideally this would be computed from layout but it is difficult to do.
     static constexpr float s_tailOcclusionAmount = 2;
+    void ClosePopupOnUnloadEvent(IInspectable const&, winrt::RoutedEventArgs e);
```

Comment:
"wrong" IInspectable type used here -- should be winrt::IInspectable. Also all winrt smart pointers should be passed by const&.
```suggestion
    void ClosePopupOnUnloadEvent(winrt::IInspectable const&, winrt::RoutedEventArgs const& e);
```